src/Libraries/LibAllowList.sol

```diff
@@ -1,66 +1,86 @@
 // SPDX-License-Identifier: LGPL-3.0-only
-/// @custom:version 1.0.0
 pragma solidity ^0.8.17;
 
 import { InvalidContract } from "../Errors/GenericErrors.sol";
+import { LibAsset } from "./LibAsset.sol";
 
-/// @title Lib Allow List
+/// @title LibAllowList
 /// @author LI.FI (https://li.fi)
-/// @notice Library for managing and accessing the conract address allow list
+/// @notice Library for managing and accessing the contract address allow list
+/// @custom:version 1.0.1
```

Comment:
I think the version bump should be bigger. Since we are changing storage layout, I would vouch for a 2.0.0.

Reply:
that's true